doc/deployment/deploy_custom.md

```diff
@@ -53,7 +413,9 @@ You may either deploy manifests you created above or edit them to customize them
 
 ### Editing your manifest to customize it further
 
-This should only be approached if you are an experienced Kubernetes administrator. 
+This functionality requires an experienced Kubernetes administrator.
+If you are attempting a highly customized deployment, 
+use one of the Pachyderm support resources below.
 
 ### Deploying
 The command you'll want to run depends on the command you ran, above.
```

Comment:
> The command you'll want to run depends on the command you ran, above. 

Does it? It's the same command, you just pass it the actual file you made. This section is literally just `kubectl apply -f ./whatever_manifest_you_created.json`. Not sure why there are 3 subsections